### PR TITLE
Use DISTFILES_MIRROR if defined to download source files

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,9 +15,15 @@ URLS := \
     http://download.qemu-project.org/qemu-$(QEMU_VERSION).tar.xz.sig \
     https://kernel.org/pub/linux/kernel/v$(firstword $(subst ., ,$(LINUX_VERSION))).x/linux-$(LINUX_VERSION).tar.sign
 
-ALL_FILES := $(addprefix dl/,$(notdir $(patsubst %.sign,%.xz,$(patsubst %.sig,%,$(URLS))) $(filter %.sig, $(URLS)) $(filter %.sign, $(URLS))))
+ALL_FILES_TMP := $(notdir $(patsubst %.sign,%.xz,$(patsubst %.sig,%,$(URLS))) $(filter %.sig, $(URLS)) $(filter %.sign, $(URLS)))
```
Comment (marmarek) [inline Makefile]: It looks like it could be specified as `$(notdir $(ALL_URLS))`. Am I missing something?

**Review by marmarek — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (fepitre) [reply Makefile]: You're right. We could simplify a bit more.

**Comment by fepitre** [comment]
All related commits for DISTFILES_MIRROR are cleaned.

**Review by fepitre — COMMENTED** [review]
Is there still anything to change? for this PR and the other one related to DISTFILES_MIRROR?

**Review by marmarek — APPROVED** [review]

**Pull request merged**